blocks/library/freeform/old-editor.js

```diff
@@ -67,6 +70,16 @@ export default class OldEditor extends Component {
 			} );
 		} );
 
+		editor.on( 'keydown', ( event ) => {
+			if ( ( event.keyCode === BACKSPACE || event.keyCode === DELETE ) &&
+					/^\n?$/.test( editor.getContent( { format: 'text' } ) ) ) {
```

Comment:
It is literally a call to body.innerText falling back to body.textContent so no it does not run the serializer. I think it should be fairly low cost.